hs_tracking/management/commands/stats.py

```diff
@@ -190,6 +191,8 @@ def yesterdays_variables(self, lookback=1):
             timestamp__gte=yesterday_start,
             timestamp__lt=today_start
         )
+        print(yesterday_start)
+        print(today_start)
```

Comment:
if you want to print these please use the logger, otherwise remove.

Reply:
Removed. It doesn't appear that these are required.